lib/rules/static-jsx.js

```diff
@@ -9,6 +9,10 @@ const {
 const eslintPluginReact = require('eslint-plugin-react')
 const curlyBracePresence = eslintPluginReact.rules['jsx-curly-brace-presence']
 
+const messages = {
+  ...curlyBracePresence.meta.messages,
+}
```

Comment:
Defining own `messages` so that we can pass current context to `jsx-curly-brace-presence` rule instance show its error messages.